.github/workflows/pr_review_checklist.yml

```diff
@@ -1,5 +1,5 @@
 name: PR Review Checklist
-on: pull_request_target
+on: pull_request
```

Comment:
Thank you for fixing my earlier error here! Since this seems to be working on PRs from forks now, can we limit this to PR creation rather than [the many default types for the pull_request event](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request)?

```
on:
  pull_request:
    types: [opened]
```